helloWorld/src/com/example/helloworld/HelloWorld.java

```diff
@@ -8,6 +10,24 @@
 
 public class HelloWorld {
     public static void main(String[] args) {
+        
+        //defining variables
+        String name;
+        
+        //add scanner
+        Scanner mrScanner = new Scanner(System.in);
+        
+        //when starting up, system will announce
         System.out.println("Hello, World!");
+        
+        //system prompt user for their name
+        System.out.println("What is your name?"):

```

Comment:
Did you test your code?

Reply:
Yes, I tested the code on intelliJ and it seemed to work. When I ran the code, the system would say "Hello, World!" and ask the user "What is your name?" then I would type in any name and the system would say "Hi, " + name

Reply:
I wonder if the code you have in intelliJ has the save revision with this one.
I don't see how this code can run. Because there is a syntax error at line #24

Reply:
Oh I see, I might have fixed it in my code but missed it in here. It shouldn't be a colon right? Should be a semi-colon

Reply:
make sure to push your code so your PR have the latest working one.
Can you add your changes?

Reply:
that makes sense! I have updated my code here to reflect what i have on my machine